Readme.md

```diff
@@ -265,6 +272,45 @@ Example:
  Console.WriteLine(result);
 ```
 
+### Working with non-normalized fractions
+
+For performance reasons, as of version 8.0.0, mathematical operations no longer automatically generate normalized fractions if one of the operands is an improper (i.e. non-normalized) fraction. This has an impact on your calculations, especially if you have used the `JsonFractionConverter` with default settings. In such cases, deserialized fractions create improper fractions, which can lead to changed behavior when calling `Equals` and `ToString`.
+
+| Symbol | Description                                                                                          |
+| ------ | ---------------------------------------------------------------------------------------------------- |
+| $NF$   | Non-normalized (possibly improper) fraction, created with `normalize: false`                         |
+| $F$    | Fraction created with `normalize: true`                                                              |
+| $⊙$   | Mathematical operation having two operands ($+$, $-$, $*$, $/$, $mod$).                              |
+
+The following rules apply:
+
+$

```

Comment:
@lipchev do you know how to add line breaks? It perfectly works locally (using VS code + markdown preview OR MathJax website). However.. not on GitHub: 

![image](https://github.com/danm-de/Fractions/assets/3825484/92201e1d-ff73-446c-8dda-4d691a58e74f)

Reply:
The AI says:

> In Markdown, you can add line breaks by ending a line with two or more spaces, then typing return. In your case, you can add line breaks to your mathematical expressions as follows:

```markdown
F ⊙ F = F  
F ⊙ NF = NF  
NF ⊙ F = NF  
NF ⊙ NF = NF  
```

Each line ends with two spaces before the return, which will create a line break.

Reply:
Oh dear. I'll probably be replaced by an AI soon. This was predicted to me at a [conference](blob:https://md-devdays.de/11865de8-717a-403c-a59d-70e8f8e22f4b) last week 🤷‍♂️.

From

![image](https://github.com/danm-de/Fractions/assets/3825484/4bd6e783-59ed-4328-9500-f95f7e4e8f09)

To

![image](https://github.com/danm-de/Fractions/assets/3825484/a7139e20-29b8-432a-9c43-830d64b8d537)